pom.xml

```diff
@@ -196,6 +195,12 @@ under the License.
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+        <groupId>org.pac4j</groupId>
+        <artifactId>pac4j-jee</artifactId>
+        <version>5.2.0</version>
+        <optional>false</optional>
+    </dependency>
```

Comment:
```suggestion
    <dependency>
      <groupId>org.pac4j</groupId>
      <artifactId>pac4j-jee</artifactId>
      <version>5.2.0</version>
    </dependency>
```

Reply:
I assume this is because `org.pac4j.core.context.JEEContext` has been moved to this module?

Reply:
yes, it is. Former `J2EContext` is now `JEEContext` and in this module